src/main.js

```diff
@@ -132,9 +132,27 @@ var load = loader({
     }),
   },
 
+  pulseClient: {
+    requires: ['cfg', 'monitor'],
+    setup: ({cfg, monitor}) => {
+      const {namespace, expiresAfter, contact} = cfg.pulse;
+      const {rootUrl, credentials} = cfg.taskcluster;
+      return new Client({
+        namespace,
+        monitor,
+        credentials: claimedCredentials({
```

Comment:
Like in tc-github, let's change this back to use `pulseCredentials` with username/password/hostname/vhost.